walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,61 @@
 
 package com.google.sps;
 
-import java.util.Collection;
+import java.util.*;
 
 public final class FindMeetingQuery {
+
+  /**
+   * Returns collection of time slots, in which we can appoint the meeting.
+   * @param events   All events in the day.
+   * @param request  Information about event that we want to add (attendees, required duration).
+   * */
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    // hash map, where key is the time moment and object is the number of attendees from the request,
+    // who will be busy or free from that time moment
+    HashMap<Integer, Long> timePoints = new HashMap<>();
+    Collection<String> attendees = request.getAttendees();
+
+    long requiredAmount = attendees.size();
```

Comment:
variable name could be more illuminating, maybe something like numberOfAttendeesRequired